opm/simulators/linalg/bda/openclKernels.hpp

```diff
@@ -0,0 +1,401 @@
+/*
+  Copyright 2020 Equinor ASA
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPENCL_HPP
+#define OPENCL_HPP
+
+namespace bda
+{
+
+    const char* kernel_1  = R"(
+    __kernel void hello1(void)
+    {
+        printf("in the kernel 1\n");
+    }
+    )";
+
+    const char* kernel_2  = R"(
+    __kernel void hello2(
+        __global int *in1,
+        __global int *in2,
+        __global int *out)
+    {
+        printf("in the kernel 2\n");
+        int idx = get_global_id(0);
+        out[idx] = in1[idx];
+        for(int i = 0; i < 20000000; ++i){
+            out[idx] = in1[idx] - in2[idx] * out[idx];
+            if(out[idx] > 100000){
+                out[idx] -= 100000;
+            }
+            if(out[idx] < -100000){
+                out[idx] += 100000;
+            }
+        }
+    }
+    )";
```

Comment:
Are we using these hello functions?

Reply:
These were used as test/debug functions, removed in 4c037b0